check-file-existence/action.yaml

```diff
@@ -0,0 +1,56 @@
+name: check-file-existence
+description: ""
+
+inputs:
+  files:
+    description: ""
+    required: true
+  condition:
+    description: ""
+    required: true
+outputs:
+  exists:
+    description: ""
+    value: ${{ steps.check-file-existence.outputs.exists }}
+
+runs:
+  using: composite
+  steps:
+    - name: File check
+      id: check-file-existence
+      run: |
+        set -e
+
+        # Split files into an array
+        tmp="${{ inputs.files }}"
+        input_files=(${tmp})
+
+        # Iterate files
+        found_files=()
+        for f in "${input_files[@]}"; do
+          if [[ -f $f ]]; then
+            echo "File exists: $f"
+            found_files+=("$f")
+          else
+            echo "File does not exist: $f"
+          fi
+        done
+
+        num_found_files=${#found_files[@]}
+        if [[ "${{ inputs.condition }}" = "or" ]]; then
```

Comment:
Just out of curiosity, do you need to use `[[ ]]` instead of `[ ]`?

Reply:
Thank you.
I prefer using `[[ ]]` when necessary because `[[ ]]` has too many features, but there seems to be no reason for rejecting it.